src/ssh/test-sshbridge.c

```diff
@@ -284,6 +287,28 @@ setup (TestCase *tc,
       g_free (content);
     }
 
+#if HAVE_SAFE_PROXY_COMMANDS
+  if (fixture && fixture->test_home_ssh_config)
+    {
+      gchar *content;
+
+      host = g_strdup ("some_host");
```

Comment:
This leaks the previous value of `host`, so it needs to be freed first. But moreover, I think the original value should be plugged into `content`, so I recommend to build `content` string first with the original `host` value, then free it, and set it to `some_host`.

Reply:
... curious that this doesn't get picked up by our `check-memory` semaphore test run. Am I missing something here?